scripts/monitornodes.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Monitor load average on nodes in a Selenium grid.
+# Runs forever, Control-C to quit.
+
+set -e
+
+if [ $# -gt 1 ]; then
+    HUB_URL=$1
+else
+    HUB_URL="http://localhost:4444/wd/hub"
+fi
+
+HUB_CONSOLE_URL="$(echo "$HUB_URL" | sed 's,/wd/hub,/grid/console,')"
+
+if [ -z "$JITSI_SSH_CONFIG" ]
+then
+    JITSI_SSH_CONFIG="$HOME/.ssh/config.d/jitsi64.config"
+fi
+
+if [ -z "$SSH" ]
+then
+    if [ -r "$JITSI_SSH_CONFIG" ]
```

Comment:
How about simply SSH_CONFIG?

Reply:
This is specifically for connecting to the nodes you're using for Jitsi, which is why I named it that way (and also because that was the name I already had for the environment variable in my local config).  Presumably other ssh will use a different config...